pkg/providers/amazon/autoscaling/autoscaling.go

```diff
@@ -97,6 +104,10 @@ func (group *Group) getSpotRequests() ([]*ec2.SpotInstanceRequest, error) {
 		return nil, err
 	}
 
+	if lc == nil {
+		return nil, nil
```

Comment:
I think it would be useful to log that no launch config is available thus we can not retrieve spot requests.